va/va_cp_vp9.h

```diff
@@ -0,0 +1,121 @@
+ * copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sub license, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ * 
+ * The above copyright notice and this permission notice (including the
+ * next paragraph) shall be included in all copies or substantial portions
+ * of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL INTEL AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+/**
+* \file va_cp_vp9.h
+* \brief The content protection API
+*
+* This file contains the \ref api_cp_vp9 "VP9 Content Protection API".
+*/
+
+#ifndef VA_CP_VP9_H
+#define VA_CP_VP9_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \defgroup api_cp_vp9 VP9 Content Protection API
+ *
+ * @{
+ */
+
+/**
+ * \brief uncompressed header for VP9 cenc decode
+ *
+ * This structure holds information for \c
+ * uncompressed_header() as defined by the VP9 specification.
+ *
+ */
+typedef struct _VACencHeaderParameterBufferVP9 {
+    
+    union {
+        struct {
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t profile                              : 2;            
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_existing_frame_flag             : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_to_show                         : 3;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t frame_type                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_frame                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t error_resilient_mode                 : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t color_space                          : 2; 
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_x                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_y                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t refresh_frame_flags                  : 1;
```

Comment:
`refresh_frame_flags` is a mask of the reference frame slots to place the current frame in - it should be a separate uint8_t variable.

Reply:
Accepted.